draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1457,9 +1455,9 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
    Historically, HTTP field values could be extended over multiple
    lines by preceding each extra line with at least one space or horizontal
    tab (<x:ref>obs-fold</x:ref>).
-   <cref>This document assumes that any such <x:ref>obs-fold</x:ref> has been replaced with one or more
+   This document assumes that any such <x:ref>obs-fold</x:ref> has been replaced with one or more
    <x:ref>SP</x:ref> octets prior to interpreting the field value,
-   as described in <xref target="line.folding"/>.</cref>
+   as described in <xref target="line.folding"/>.
```

Comment:
Note that this is still referencing obs-fold as an ABNF target. It would be better to replace that with "obsolete line folding". I.e.,

```
   Historically, HTTP/1.x field values could be extended over multiple
   lines by preceding each extra line with at least one space or horizontal
   tab (obsolete line folding). This document assumes that obsolete line folding
   is removed by the HTTP/1.x message parsing algorithm (<xref target="line.folding"/>)
   prior to interpreting the field value.
```